src/web/lib/components/AppHeader/index.scss

```diff
@@ -57,12 +57,21 @@
     padding: 0;
     width: 64px;
 
+    span {
+      pointer-events: none;
+    }
+
     .app-header__button-icon {
       align-items: Center;
       display: flex;
       height: 20px;
       margin-bottom: 1px;
       width: 22px;
+      pointer-events: none;
+
+      img {
+        pointer-events: none;
```

Comment:
This was not necessary, as `pointer-events` makes an element and all of its descendants unclickable. (The parentt elemen (`.app-header__button-icon`) was already assigned `pointer-events`).